google/cloud/datacatalog_v1beta1/gapic/policy_tag_manager_client.py

```diff
@@ -420,9 +456,8 @@ def update_taxonomy(
 
     def list_taxonomies(
         self,
-        parent=None,
+        parent,
         page_size=None,
-        page_token=None,
```

Comment:
Potentially problematic that this is being removed. (Although this is in the beta surface, so technically allowed)

Reply:
Looks like the gapic config missed configuring pagination for this method, so this should actually be an improvement. https://github.com/googleapis/googleapis/blob/master/google/cloud/datacatalog/v1beta1/datacatalog_gapic.legacy.yaml